twitter1.css

```diff
@@ -0,0 +1,299 @@
+* {
```

Comment:
Personally I wouldn't apply styles using the `*`. The reason being is that some elements have `margin`/`padding` by default. By removing these properties on every element, you will have to re-add them when you want them. 

This is a personal preference though, so if you prefer to do it that way that is fair enough :)